docusaurus/docs/key-concepts/plugin-types-usage.md

```diff
@@ -30,7 +30,7 @@ Grafana plugin development allows for many options depending on the type of user
 - **Data-source plugin** - a connection to a new database or other source of data.
 - **App plugin** - an integrated out-of-the-box experience.
 
-Refer to [Get started](../get-started/get-started.mdx) for instructions on how to quickly scaffold [each type](https://grafana.com/developers/plugin-tools/reference/prompts.md/#what-type-of-plugin-would-you-like) of plugin.
+Refer to [Get started](../get-started/get-started.mdx) for instructions on how to quickly scaffold [each type](https://grafana.com/developers/plugin-tools/reference/prompts#select-a-plugin-type) of plugin.
```

Comment:
Thanks for spotting these issues. We should replace this (and all the other links in this PR) with relative links instead of these absolute ones. Could you please update your PR with those?

Reply:
Thanks for the review and suggestion, @academo ! Will do!